lib/connection/process.coffee

```diff
@@ -73,12 +100,8 @@ module.exports = jlprocess =
     return if @proc?
     client.booting()
 
-    @checkPath(@jlpath()).then (exists) =>
-      if not exists
-        @jlNotFound @jlpath()
-        client.cancelBoot()
-        return
-
+    @checkPath(@jlpath())
+    .then =>
```

Comment:
We usually indent the `.then` and `.catch` to make it clear it's not a new statement